tests/integration/docusaurus/expectations/creating_custom_expectations/expect_table_columns_to_be_unique.py

```diff
@@ -0,0 +1,226 @@
+"""
+This is a template for creating custom TableExpectations.
+For detailed instructions on how to use it, please see:
+    https://docs.greatexpectations.io/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_table_expectations
+"""
+
+from typing import Dict, Optional
+
+import pandas as pd
+
+from great_expectations.core.expectation_configuration import ExpectationConfiguration
+from great_expectations.exceptions import InvalidExpectationConfigurationError
+from great_expectations.execution_engine import (
+    ExecutionEngine,
+    PandasExecutionEngine,
+    SparkDFExecutionEngine,
+    SqlAlchemyExecutionEngine,
+)
+from great_expectations.expectations.expectation import TableExpectation
+from great_expectations.expectations.metrics.metric_provider import (
+    MetricConfiguration,
+    MetricDomainTypes,
+    metric_value,
+)
+from great_expectations.expectations.metrics.table_metric_provider import (
+    TableMetricProvider,
+)
+
+
+# <snippet>
+# This class defines a Metric to support your Expectation.
+class TableColumnsUnique(TableMetricProvider):
+
+    # This is the id string that will be used to reference your Metric.
+    metric_name = "table.columns.unique"
+
+    # This method implements the core logic for the PandasExecutionEngine
+    @metric_value(engine=PandasExecutionEngine)
+    def _pandas(
+        cls,
+        execution_engine,
+        metric_domain_kwargs,
+        metric_value_kwargs,
+        metrics,
+        runtime_configuration,
+    ):
+        df, _, _ = execution_engine.get_compute_domain(
+            metric_domain_kwargs, domain_type=MetricDomainTypes.TABLE
+        )
+
+        unique_columns = set(df.T.drop_duplicates().T.columns)
+
+        return unique_columns
+
+    # @metric_value(engine=SqlAlchemyExecutionEngine)
```

Comment:
This commented out code will fall out of date; either remove it or create a different file where it can be tested.

Reply:
My bad; I think these shouldn't be commented out. Our pattern traditionally has been to leave these in with `Raise NotImplementedError`